### PR TITLE
Monitoring Alerting and Centralized Logging Integration Page

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
++++
+title = "Chef Automate HA integrations"
```
Comment (karunesh-tech) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:2]: @dishanktiwari2501 
Please update all the techincal blog links should be actual public facing URLs and not the internal sharepoint link, please replace in all the 4 sections

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- Datadog
+- AWS CloudWatch
+- Prometheus
+- Elastic Stack (ELK)
+
+## Datadog
+
+Datadog is a monitoring and analytics platform that delivers real-time insights into application, infrastructure, and cloud service performance. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and DataDog Integration](https://github.com/chef/monitoring-integration-automate/tree/main/data-dog)
```
Comment (karunesh-tech) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:28]: @dishanktiwari2501  please update the data dog whitepaer link with whitepaper data dog section link, which is:
https://github.com/chef/monitoring-integration-automate/blob/main/Whitepaper_AutomateHA_Monitoring_and_Alerting.md#datadog-integration-with-automate-ha---monitoring

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- [Technical Blog reference of Automate HA and DataDog](https://progresssoftware-my.sharepoint.com/:w:/g/personal/apravati_progress_com/Eb2zXjNHa71Brszy3V_SqsoB0LtyGP7eaBXtBlaqj9rGSA?e=gJcC8b)
+
+## AWS CloudWatch
+
+Chef Automate HA with AWS CloudWatch gives automate and compliance teams the necessary redundancy and visibility to successfully operate the solution. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and CloudWatch Integration](https://github.com/chef/monitoring-integration-automate/tree/main/cloud-watch)
```
Comment (karunesh-tech) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:35]: @dishanktiwari2501  please update the data dog whitepaer link with whitepaper data dog section link, which is:
https://github.com/chef/monitoring-integration-automate/blob/main/Whitepaper_AutomateHA_Monitoring_and_Alerting.md#cloudwatch-integration-with-automate-ha---monitoring

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- [Technical Blog reference of Automate HA and CloudWatch](https://progresssoftware-my.sharepoint.com/:w:/g/personal/apravati_progress_com/EQsURaeVai5Eszc9-3yiHWEBKCeDV0Nz-LhqC2giqefO-Q?e=KM0iWB)
+
+## Prometheus
+
+Prometheus, an open-source tool excels at scraping metrics from HTTP endpoints, making it a go-to for harvesting data from various targets. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and Prometheus Integration](https://github.com/chef/monitoring-integration-automate/tree/main/prometheus)
```
Comment (karunesh-tech) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:42]: @dishanktiwari2501 please update the data dog whitepaer link with whitepaper data dog section link, which is:
https://github.com/chef/monitoring-integration-automate/blob/main/Whitepaper_AutomateHA_Monitoring_and_Alerting.md#prometheus-integration-with-automate-ha---monitoring

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+- [Technical Blog reference of Automate HA and Prometheus](https://progresssoftware-my.sharepoint.com/:w:/g/personal/apravati_progress_com/EZ23D4XeuKJGu-yMzmIehfwBPmTouhykSFqRhCJXdWSQWg?e=tysXun)
+
+## Elastic Stack
+
+The ELK stack is a trio of open-source tools that have revolutionized how businesses harness their data. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and ELK Integration](https://github.com/chef/monitoring-integration-automate/tree/main/ELK)
```
Comment (karunesh-tech) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:49]: @dishanktiwari2501 please update the data dog whitepaer link with whitepaper data dog section link, which is:
https://github.com/chef/monitoring-integration-automate/blob/main/Whitepaper_AutomateHA_Monitoring_and_Alerting.md#elk-integration-with-automate-ha---centralised-logging

**Review by karunesh-tech — COMMENTED** [review]
@dishanktiwari2501 please lookinto the review comments, mostly are link updates and make the changes accordingly

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Integrations"
+    parent = "automate/deploy_high_availability"
+    identifier = "automate/deploy_high_availability/monitoring-alerting-centralized-logging-integration.md Chef Automate HA integrations"
+    weight = 75
++++
+
+You can integrate third-party platforms with [Chef Automate HA](/automate/ha) to monitor Autoamte's health, infrastructure, components, and services.
```
Comment (karunesh-tech) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:15]: Pls correct Automate spelling here

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
++++
```
[the remark this answers is not in the document]
Comment (karunesh-tech) [reply components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:1]: @dishanktiwari2501  pls add this in note section as requested

[the remark this answers is not in the document]
Comment (IanMadd) [reply components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:1]: No notes. The version number is a prerequisite and customers know that they can get in touch with customer support for more information. 

The supported version should be listed as 

```
## Prerequisites

These integrations are supported on Chef Automate HA x.y.
```

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+Chef Automate HA integrates with the following platforms:
+
+- AWS CloudWatch
+- Datadog
+- Elastic Stack (ELK)
+- Prometheus
+
+## Prerequisites
+
+These integrations are supported on Chef Automate HA 4.12.66.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:26]:
```suggestion
These integrations are supported on Chef Automate HA 4.12.66 and above.
```

File: components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Integrations"
+    parent = "automate/deploy_high_availability"
+    identifier = "automate/deploy_high_availability/monitoring-alerting-centralized-logging-integration.md Chef Automate HA integrations"
+    weight = 75
++++
+
+You can integrate third-party platforms with [Chef Automate HA](/automate/ha) to monitor Autoamte's health, infrastructure, components, and services.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md:15]:
```suggestion
You can integrate third-party platforms with [Chef Automate HA](/automate/ha) to monitor Automate's health, infrastructure, components, and services.
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**